decksite/achievements.py

```diff
@@ -0,0 +1,237 @@
+from typing import TYPE_CHECKING, Dict, List, Optional
+
+from flask import url_for
+from flask_babel import ngettext
+
+import decksite
+from decksite.data import query
+from magic import tournaments
+
+if TYPE_CHECKING:
+    from decksite.data import person # pylint:disable=unused-import
+# Disabling unused-import supposedly not needed here but actually seems to be?
+
+def load_query(people_by_id: Dict[int, 'person.Person'], season_id: Optional[int]) -> str:
+    columns = ', '.join(f'SUM({a.key}) as {a.key}' for a in Achievement.all_achs if a.in_db)
```

Comment:
nit. It wouldn't kill you to say all_achievements :D thought you were referring to archetypes for a second.